CHANGELOG.md

```diff
@@ -206,3 +206,10 @@
 ## [v2.21.2](https://github.com/BoltApp/bolt-magento2/releases/tag/2.21.2) 2021-05-10
 - Fix bug with backoffice orders
 - Fix a regression with handling custom fields
+## [v2.22.0](https://github.com/BoltApp/bolt-magento2/releases/tag/2.22.0) 2021-05-14
```

Comment:
Target date should probably be sometime next week, no?

Reply:
I just put it as today since this file will probably change between now and then. I can move it, I believe QA has it due by thursday on their board.